ledger-api/rs-grpc-akka/src/main/scala/com/digitalasset/grpc/adapter/AkkaExecutionSequencerPool.scala

```diff
@@ -34,7 +33,7 @@ class AkkaExecutionSequencerPool(
 
   def closeAsync(): Future[Unit] = {
     implicit val ec: ExecutionContext = system.dispatcher
-    val eventuallyClosed: Future[Seq[Done]] = Future.sequence(pool.map(_.closeAsync)(breakOut))
+    val eventuallyClosed: Future[Seq[Done]] = Future.sequence(pool.map(_.closeAsync).toSeq)
```

Comment:
This probably needs a `.view` to emulate the original behavior of not constructing an intermediate collection.

```suggestion
    val eventuallyClosed: Future[Seq[Done]] = Future.sequence(pool.view.map(_.closeAsync).toSeq)
```